src/pages/__tests__/SignIn.test.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+
+import App from 'components/App';
```

Comment:
We can to that but with testingRender.tsx we avoid that kind of approaches, it provides all the possibilities to test pages by path and it is also mocking the store:

<br />
![image](https://user-images.githubusercontent.com/41349687/163869940-9fb3b507-96d8-4a33-b5af-662997dd12b1.png)

thats why i'm rendering app component but i'm testing sign in feature, because i used sign in path so it means my test is gonna render sign in component :)